crates/turbopack-ecmascript/src/lib.rs

```diff
@@ -84,13 +84,17 @@ use crate::{
 };
 
 #[turbo_tasks::value(serialization = "auto_for_input")]
-#[derive(PartialOrd, Ord, Hash, Debug, Default, Copy, Clone)]
+#[derive(PartialOrd, Ord, Hash, Debug, Copy, Clone)]
 pub struct EcmascriptOptions {
     /// module is split into smaller module parts which can be selectively
     /// imported
     pub split_into_parts: bool,
     /// imports will import parts of modules
     pub import_parts: bool,
+
+    pub transforms: EcmascriptInputTransformsVc,
+
+    pub compile_time_info: CompileTimeInfoVc,
```

Comment:
I'm not sure if this can be changed to `Option`, but there's one problem if I don't